webpack.config.js

```diff
@@ -1,105 +1,48 @@
 const path = require("path");
 const webpack = require("webpack");
-const CopyWebpackPlugin = require("copy-webpack-plugin");
-const HtmlWebpackPlugin = require("html-webpack-plugin");
-const CleanWebpackPlugin = require("clean-webpack-plugin");
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
-const babelConfig = require("./babel.config");
 
 // absolute path disables recursive module resolution, so build a relative one
-const nodedir = path.relative(process.cwd(), path.resolve(process.env.SRCDIR || __dirname, "node_modules"));
-
-const [mode, devtool] =
```

Comment:
This removal is tracked here: https://github.com/osbuild/cockpit-composer/issues/1576